build/cmake/unabto_project.cmake

```diff
@@ -117,6 +117,7 @@ if (NOT UNABTO_DISABLE_EPOLL)
   CHECK_INCLUDE_FILES("sys/epoll.h" UNABTO_HAVE_EPOLL)
 endif()
 
+list(APPEND unabto_src ${unabto_stateful_push_service_src})
```

Comment:
probably not a good idea to include the stateful push service in all unabto projects.